resources/migrations/001_update_migrations.yaml

```diff
@@ -4381,6 +4390,380 @@ databaseChangeLog:
+            columnName: is_sandboxed
+            newDataType: ${boolean.type}
+      rollback:
+        - modifyDataType:
+            tableName: query_execution
+            columnName: is_sandboxed
+            newDataType: boolean
+      preConditions:
+        - onFail: MARK_RAN
+        - dbms:
+            type: mysql,mariadb
+
+  - changeSet:
+      id: v49.00-029
+      author: noahmoss
+      comment: >-
+        Added 0.49.0 - modify type of view_log.has_access
+        from boolean to ${boolean.type} on mysql,mariadb
+      dbms: mysql,mariadb
+      changes:
+        - modifyDataType:
+            tableName: view_log
+            columnName: has_access
+            newDataType: ${boolean.type}
+      rollback:
+        - modifyDataType:
+            tableName: view_log
+            columnName: has_access
+            newDataType: boolean
+      preConditions:
+        - onFail: MARK_RAN
+        - dbms:
+            type: mysql,mariadb
+
+  - changeSet:
+      id: v49.00-030
+      author: noahmoss
+      comment: >-
+        Added 0.49.0 - modify type of metabase_field.database_indexed
+        from boolean to ${boolean.type} on mysql,mariadb
+      dbms: mysql,mariadb
+      changes:
+        - modifyDataType:
+            tableName: metabase_field
+            columnName: database_indexed
+            newDataType: ${boolean.type}
+      rollback:
+        - modifyDataType:
+            tableName: metabase_field
+            columnName: database_indexed
+            newDataType: boolean
+      preConditions:
+        - onFail: MARK_RAN
+        - dbms:
+            type: mysql,mariadb
+
+  - changeSet:
+      id: v49.00-031
+      author: noahmoss
+      comment: >-
+        Added 0.49.0 - modify type of metabase_table.database_require_filter
+        from boolean to ${boolean.type} on mysql,mariadb
+      dbms: mysql,mariadb
+      changes:
+        - modifyDataType:
+            tableName: metabase_table
+            columnName: database_require_filter
+            newDataType: ${boolean.type}
+      rollback:
+        - modifyDataType:
+            tableName: metabase_table
+            columnName: database_require_filter
+            newDataType: boolean
+      preConditions:
+        - onFail: MARK_RAN
+        - dbms:
+            type: mysql,mariadb
+
+  - changeSet:
+      id: v49.00-032
+      author: noahmoss
+      comment: >-
+        Added 0.49.0 - modify type of DATABASECHANGELOGLOCK.LOCKED
+        from boolean to ${boolean.type} on mysql,mariadb
+      dbms: mysql,mariadb
+      changes:
+        - modifyDataType:
+            tableName: DATABASECHANGELOGLOCK
+            columnName: LOCKED
+            newDataType: ${boolean.type}
+      rollback:
+        - modifyDataType:
+            tableName: DATABASECHANGELOGLOCK
+            columnName: LOCKED
+            newDataType: boolean
+      preConditions:
+        - onFail: MARK_RAN
+        - dbms:
+            type: mysql,mariadb
```

Comment:
Why is this migration needed? Do we care about this type? `docs/installation-and-operation/migrating-from-h2.md` instructs users to use `BOOLEAN`.